gallery/how_to/compile_models/from_keras.py

```diff
@@ -76,7 +77,7 @@
 # A single cat dominates the examples!
 from PIL import Image
 from matplotlib import pyplot as plt
-from keras.applications.resnet50 import preprocess_input
+from tensorflow.keras.applications.resnet50 import preprocess_input
 
```

Comment:
I don't get why I had to make this change. After we upgraded to keras 2.6 in https://github.com/apache/tvm/pull/10084, `keras.applications` no longer exists. The doc build runs on the gpu image, which should have been updated to keras 2.6 in https://github.com/apache/tvm/pull/10654. So this error should have been caught during the update of https://github.com/apache/tvm/pull/10654.

cc @driazati @leandron

Reply:
also not sure, the mapping of image version -> range of commits is very difficult to track down (but should be fixed by #10646)